py/STREAM/Settings/ConfinementTime.py

```diff
@@ -0,0 +1,45 @@
+from DREAM.Settings.Equations.EquationException import EquationException
+from DREAM.Settings.Equations.PrescribedParameter import PrescribedParameter
+from DREAM.Settings.Equations.PrescribedInitialParameter import PrescribedInitialParameter
+from DREAM.Settings.Equations.PrescribedScalarParameter import PrescribedScalarParameter
+from DREAM.Settings.Equations.UnknownQuantity import UnknownQuantity
+
+DEFAULT_TYPE = 1
+DEFAULT_MIXTE = 0
```

Comment:
MIXTE: define: not clear from name of variable?

Reply:
perhaps using the wording 'max_perp_tau' is more appropriate? If too much work we can leave this as is

Reply:
Yes it is I m going to make the modifications during the afternoon

Reply:
this was resolved with maxPerpLaw in new version. Closing issue.